packages/maven/sentry-android/2.0.0.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Android",
+  "canonical": "io.sentry:sentry-android",
```

Comment:
canonical has to have some indication of the package manager.
I.e: 

https://github.com/getsentry/sentry-release-registry/blob/5b94f1d35771b9e667329b69ddc608b2fa3ff3e5/packages/nuget/Sentry.AspNetCore/2.0.3.json#L3

https://github.com/getsentry/sentry-release-registry/blob/5b94f1d35771b9e667329b69ddc608b2fa3ff3e5/packages/npm/%40sentry/browser/4.1.1.json#L3

```suggestion
  "canonical": "maven:sentry-android",
```

or: `"canonical": "maven:io.sentry.sentry-android",`

And in the SDK we'll need to change this:
https://github.com/getsentry/sentry-android/blob/16838d4330aab86566f63a09c6db8c9f3390bd76/sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java#L230

To match up.

@untitaker wrote the feature to suggest the new SDK version, does that match up?

Reply:
https://github.com/getsentry/sentry-android/pull/269

Reply:
Yup that sounds right!

Reply:
@bruno-garcia so we will need to version all the packages and not only `sentry-android`, right?
like sentry-android-core, sentry-android-ndk, sentry-core...

Reply:
exactly. 
Since we don't have Android docs on this list 
![image](https://user-images.githubusercontent.com/1633368/74546361-79d47e80-4f18-11ea-8a6a-00b01070b05b.png)

Which I believe was an oversight, adding it here won't make any difference besides hint in Sentry to update to a new SDK version.

Maybe we just close this PR and focus on craft instead?

Reply:
agreed, let's move Craft forward and then this will be done by Craft anyway.